Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_set.py

```diff
@@ -172,12 +172,13 @@ def match_each(self, extractor: DateTimeExtractor, source: str, reference: datet
 
         for match in regex.finditer(self.config.set_week_day_regex, source):
             trimmed_source = source[0:match.start(
-            )] + RegExpUtility.get_group(match, 'weekday') + source[match.end():]
+            )] + RegExpUtility.get_group(match, Constants.WEEKDAY_GROUP_NAME) + source[match.end():]
```

Comment:
Can we fix this line? Should be better to open and close the source brackets at the same line; like this source[0:match.start()]